app/boards/shields/a_dux/boards/stemcell.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/ {
+	chosen {
+		zmk,split-serial = &usart1;
+	};
+};
+
+&usart1 {
+	status = "okay";
+};
+
+&kscan0 {
```

Comment:
Why would the pins for a kscan device need overriding for just one board?

Reply:
Existing split keyboards use any one of four ports D0, D1, D2 and D3 for split communication
Ex: Lily58 uses D2 (promicro RX pin), A_Dux uses D1 (I2C SDA pin) etc

STM32 chips support half duplex UART comm only on TX pins. 
So STeMCell has jumpers on back side to swap the TX/RX pins.
This depends completely on the shield which uses the pin for communication.

Other way is to use some KConfig variable to detect and swap pins. 
But i have not figured out how to do it exactly using Kconfig and device tree yet,

So in the meantime, i have just juggling the pins inside the shields directly.